modules/homePage/component/v2/SecondJumbotron/Section.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ButtonSecondary from "../../../../../uikit/buttonV2/ButtonSecondary";
+
+function SecondJumbotronSection({ children }: { children: React.ReactNode }) {
+    return <div className="grid grid-cols-1 text-center text-white md:grid-cols-2 md:text-left ">{children}</div>;
+}
+
+function SecondJumbotronTitle({ children }: { children: React.ReactNode }) {
+    return <h2 className="mt-10 text-3xl font-extrabold leading-relaxed md:text-5xl md:leading-[56px]">{children}</h2>;
```

Comment:
All `text styles` has already been configured in `globals.css`, for example in this `SecondJumbotronTitle` we can use `heading-h3 md:heading-h2` in className.

Reply:
I see
we need to write docs about this 🤣

Reply:
Oh yess, it has already been documented on figma by @ayikfour . You can press the background on figma and select the inspect tab sir 👍
![image](https://user-images.githubusercontent.com/81855912/171093578-5e0ccc47-0e9b-46d8-b2ce-e50d4901a2a2.png)

Reply:
updated ser